flink-runtime/src/main/java/org/apache/flink/runtime/dispatcher/Dispatcher.java

```diff
@@ -100,8 +97,6 @@
 
 	private final LeaderElectionService leaderElectionService;
 
-	private final JobExecutionResultCache jobExecutionResultCache = new JobExecutionResultCache();
```

Comment:
`JobExecutionResultCache` and `JobExecutionResultCacheTest`  should be also removed in this commit

Reply:
Forgot about this. Will remove it.